x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/Constants.java

```diff
@@ -358,6 +358,7 @@ public class Constants {
         "cluster:monitor/nodes/data_tier_usage",
         "cluster:monitor/nodes/features",
         "cluster:monitor/nodes/hot_threads",
+        "cluster:monitor/nodes/index_mode_stats",
```

Comment:
Note: This action seems to be [only existing in 8.17.+ versions](https://github.com/elastic/elasticsearch/pull/116128). We should not be backporting it to 8.16.2.